scripts/benchmarks/parse-benchmarks-data.js

```diff
@@ -86,7 +99,7 @@ if (benchmarkYear && performanceYear) {
   process.stdin.on('end', function() {
     // Quote option update to handle 2020 data
     const columns = benchmarkType === 'MCC' ? MCC_BENCHMARK_CSV_COLUMNS : BENCHMARK_CSV_COLUMNS;
-    parse(benchmarksData, {columns, from: 3, quote: '"'}, function(err, records) {
+    parse(benchmarksData, {columns, from: 2, quote: '"'}, function(err, records) {
```

Comment:
what is the reason for this change?

Reply:
I'm guessing in earlier years there were two lines of headers. But with this CSV, the data starts on row 2. I was seeing the first row being ignored until I changed that.